### PR TITLE
Create events page

### PR DESCRIPTION
**Pull request by zianc**
This PR creates an events page for viewing SWE's current events. Events are
pulled from SWE's Facebook page and are displayed by weeks within a quarter.
Lazy loading assures that only a certain number of events are displayed on the
initial load.

Tasks in progress include redesigning the site to be more mobile-friendly. The
current UI is attached below. The primary task is incorporating a check-in
system.
Screen Shot 2020-10-17 at 12 11 04 AM

### CODE REVIEW

**Comment by nikhitashanker** [comment]
> This PR creates an events page for viewing SWE's current events. Events are
> pulled from SWE's Facebook page and are displayed by weeks within a quarter.
> Lazy loading assures that only a certain number of events are displayed on the
> initial load.
> 
> Tasks in progress include redesigning the site to be more mobile-friendly.

If possible add a screenshot of current setup.

File: server/routes/events.js [hunk 1]
```diff
@@ -73,20 +103,22 @@ router.post('/', function(req, res, next) {
   	fb_id: req.body.event_id, 
   	name: req.body.name, 
   	starts_at: req.body.starts_at, 
-  	ends_at: req.body.ends_at,
-  	quarter: req.body.quarter,
+    ends_at: req.body.ends_at,
+    attendance_code: req.body.attendance_code,
+  	period: req.body.period,
```
Comment (nikhitashanker) [inline server/routes/events.js:108]: Format strange.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,406 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (nikhitashanker) [inline yarn.lock:1]: Important: Please remove this file from the PR before merging from master.

**Review by nikhitashanker — APPROVED** [review]
Please add the issues and link issues to milestones, so we can practice the workflow!

**Review by christinatong01 — APPROVED** [review]

**Pull request merged**